index.html

```diff
@@ -167,7 +167,35 @@ <h6>LET'S CONNECT</h6>
                             <p>skills</p>
                             <i class="fa fa-angle-right"></i>
                         </li>
-                    </ul>   
+                    </ul>  
+        </section>
+        <section class="contact">
+            <div class="contact-form">
+                <h2>Contact Me</h2>
+                <p>If you have an application you are interested in developing, 
+                    a feature that you need built or a project that needs coding. 
+                    I'd love to help with it 
+                </p>
+                <form action="https://formspree.io/f/mdobaojq" method="POST">
+                    <ul class="form">
+                        <li class="form-content">
+                        <label for="name"></label>
+                        <input type="text" id="name" placeholder="Enter Your Name" maxlength="30" required>
+                        </li>
+                        <li class="form-content">
+                            <label for="email"></label>
+                            <input type="email" id="email" placeholder="youremail@gmail.com" required>
```

Comment:
- [ ] As I mentioned earlier, you need the `name` attribute here as well as this is required for fromspree to work.

_Note: You don't need to add the `name` attribute to the `textarea`, it is only mandatory for name and email input fields._